components/authorization/kcp/spi-secrets.yaml

```
@@ -0,0 +1,30 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: spi-secrets
+  namespace: spi-system
+rules:
+  - verbs:
+      - create
+      - delete
+      - list
+      - get
+      - update
+    apiGroups:
+      - ''
+    resources:
+      - secrets
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: spi-secrets-rolebinding
+  namespace: spi-system
+subjects:
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: rh-sso:skabashn
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: spi-system
```

the name should match the name of the Role, not namespace, right?
```suggestion
  name: spi-secrets
```